### DISCUSSION TITLE
What is the best way to fill a select

### DISCUSSION THREAD

**Discussion by rabol** [bladewindui/ui#68] [Q&A]
Hi

I was wondering how to best fill a 'drop-down'
as far as I can see one have to 'manipulate' the result of this will not work.

```
Users::all()->pluck('name', id')->toArray();
```
as it does not have a key/value that is supported by:
```
<x-bladewind.dropdown>
```

so what is the best way ?

**Comment by mkocansey** [comment]
Hello @rabol ,

The Bladewind Dropdown actually allows you to specify which keys in your array you want to use as the label and corresponding value.

In your case we will have

`$users = Users::all()->pluck('name', id')->toArray();`

`<x-bladewind.dropdown data="{{ json_encode($users) }}" label_key="name" value_key="id">`

Hope this helps though its been a couple of months before responding. Apologies for that.
